src/mpi/init/init_gpu.c

```diff
@@ -26,6 +26,7 @@ int MPII_init_gpu(void)
         MPL_gpu_info.specialized_cache = specialized_cache;
         MPL_gpu_info.use_immediate_cmdlist = MPIR_CVAR_GPU_USE_IMMEDIATE_COMMAND_LIST;
         MPL_gpu_info.roundrobin_cmdq = MPIR_CVAR_GPU_ROUND_ROBIN_COMMAND_QUEUES;
+        MPL_gpu_info.max_cache_entries = MPIR_CVAR_CH4_IPC_GPU_MAX_CACHE_ENTRIES;
```

Comment:
Since they are accessed outside ch4, the CVAR name shouldn't use the `CH4` namespace. I see there are other CVARs with the same naming issues, so we may merge this PR as is and refactor later.

Please change the commit message subject since it changes files outside `mpl`.

Reply:
Reworded both cvar commit messages